gradle/libs.versions.toml

```diff
@@ -51,7 +51,7 @@ androidAnnotations = "androidx.annotation:annotation:1.1.0"
 playServicesMaps = "com.google.android.gms:play-services-maps:17.0.0"
 
 officialKubernetesClient = "io.kubernetes:client-java:8.0.0"
-kubernetesClient = "io.fabric8:kubernetes-client:4.9.0"
+kubernetesClient = "io.fabric8:kubernetes-client:5.5.0"
```

Comment:
For the record: https://github.com/fabric8io/kubernetes-client#compatibility-matrix
4.9.0 has the same minimum k8s supported version as 5.5.0